### ISSUE TITLE
Provide an example that includes `$` in the replacement string. 

### ISSUE THREAD

**Issue by sheki** [facebookincubator/fastmod#15]
I am working with some hack/php code and it has a lot of `$` symbols. I have not found a way to include `$` in the replacement code.

The below command 
```
fastmod 'function unfurl_(\w+)\(.*message.*\)' 'function unfurl_${1}(team_t \$team, \$item, UnfurlContext $context)' file.php
```
Produces this

```
- function unfurl_unknown(team_t $team, $item, $message, bool $do_link_unfurling, bool $do_media_unfurling): legacy_ret_t {
+ function unfurl_unknown(team_t \, \, UnfurlContext ): legacy_ret_t {
```

**Comment by swolchok** [comment]
https://docs.rs/regex/1.2.0/regex/struct.Regex.html#replacement-string-syntax says "to write a literal `$`, use `$$`." I'll call this out in our docs because it's not on the main page of the regex crate documentation. Thanks!

**Comment by swolchok** [comment]
Fixed by d9d917ec9907f6f6b71cce2e5fde0659ac1023b4
